SIT.Manager/Services/AppUpdaterService.cs

```diff
@@ -97,26 +97,31 @@ private async Task ExtractUpdatedManager(string zipPath, string destination)
 
     public async Task<bool> CheckForUpdate()
     {
-        if (_managerConfigService.Config.LookForUpdates)
+        Version currentVersion = Assembly.GetEntryAssembly()?.GetName().Version ?? new Version("0");
+        Version latestVersion = new();
+
+        if (_managerConfigService.Config.LookForUpdates && DateTime.Now.AddHours(-1) < _managerConfigService.Config.LastManagerUpdateCheckTime)
```

Comment:
This code kinda threw me through a bit of a mental loop for a moment. Could we instead do something like `DateTime.Now - _managerConfigService.Config.LastManagerUpdateCheckTime > TimeSpan.FromHours(1)`?

Reply:
Ye it makes me have to think every time but we're looking for when the last update time is over an hour ago and I'm not sure of a more obvious way of doing it